src/mixxx.h

```diff
@@ -171,6 +174,7 @@ class MixxxMainWindow : public QMainWindow {
 
     void linkSkinWidget(ControlObjectSlave** pCOS,
                         ConfigKey key, const char* slot);
+    QString getTitleFromTrack(TrackPointer pTrack);
```

Comment:
c++!  if you declare a function in a header, but never use it, the compiler never has a chance to discover it doesn't exist.